### PR TITLE
Update Masca 1.1.0

### PR DESCRIPTION
**Pull request by Vid201**
This PR adds a new version of Masca Snap (v1.1.0).

Changes in the new version can be found here: https://hackmd.io/eOilSHWdS7uylpdHBkxN3A

### CODE REVIEW

**Comment by Montoya** [comment]
This closes #280

**Review by Montoya — APPROVED** [review]

**Comment by Montoya** [comment]
I reviewed the change @Mrtenz, just needs codeowner approval now

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**
